contracts/ZilCrossChainManager.scilla

```diff
@@ -13,6 +13,7 @@ let one_msg =
 
 let bool_active = True
 let bool_inactive = False
+let unit = Unit
```

Comment:
The standard library `Polynetwork` already defines this constant.
```suggestion
let unit = Unit
```

Reply:
okie